### ISSUE TITLE
OnePlus with dotOS error “An error occurred downloading the APK, Please try again”

### ISSUE THREAD

**Issue by HuaiChunHao** [KieronQuinn/AmbientMusicMod#127]
![Screenshot_20221217-214730_Ambient Music Mod](https://user-images.githubusercontent.com/105863453/208245135-4ac588ae-547e-4859-8c45-d3c997984e2b.png)
![Screenshot_20221217-214750_设置](https://user-images.githubusercontent.com/105863453/208245139-4cb4c7c1-c47f-4665-8e71-fed3ed0c6eb5.png)

Phone model = `"OnePlus Ace Race(12+256)"`
Region = `"China Mainland"`
Proxy = `true`

I can use Google Services and Google Play . Root Environment is `Riru-Sui`
How can I solve it? Is there a way without downloading it through App? That is, I manually downloaded it and imported it into this App.

Thank you !

**Comment by KieronQuinn** [comment]
It's trying to download from GitHub, that error is coming from the Download Manager. As you're in China, is it possible it's getting blocked?

**Comment by HuaiChunHao** [comment]
> It's trying to download from GitHub, that error is coming from the Download Manager. As you're in China, is it possible it's getting blocked?

When I finished posting this Issue last night, it miraculously started downloading! I don't know what kind of magic this is, I've tried proxy bypass gateways, Surfboard proxies, and a mix of them before. All three methods allow me to connect to GitHub, but not download. However, last night (maybe eight hours ago for you) it miraculously started downloading, thank you very much.
Thanks to the project authors for their efforts, this is really a good project, thank you.
